### PR TITLE
Added Summary View Config Plus Newline

### PR DESCRIPTION
**Pull request by tharmaman**
Basically enables us to pass in DAOList view to render for notifications instead of a ScrollTableView.

### CODE REVIEW

**Review by nanoscott — APPROVED** [review]

File: src/foam/comics/v2/DAOControllerConfig.js [hunk 1]
```diff
@@ -61,6 +61,20 @@ foam.CLASS({
       name: 'browseSubtitle',
       expression: function(of) { return "View all " + of.model_.plural.toLowerCase() +'.' }
     },
+    {
+      class: 'foam.u2.ViewSpecWithJava',
```
Comment (kgrgreer) [inline src/foam/comics/v2/DAOControllerConfig.js:65]: Why is this needed? Can't we just refine rather than extend ViewSpec and then just use the regular ViewSpec?
